app/javascript/forms/mappers/formsFieldsMapper.jsx

```diff
@@ -12,6 +24,9 @@ const fieldsMapper = {
   'validate-credentials': AsyncCredentials,
   'credentials-password-edit': EditSecretField,
   'secret-switch-field': SecretSwitchField,
+  wizard: MiqWizard,
+  'provider-initial-step': InitialWizardStep,
+  summary,
```

Comment:
`summary` is not a good generic name for a div with a title and some specific json.

Since this would affect any new form, maybe this should be JsonSummary or something?

Reply:
ah this is left over it will be deleted. First iteration was in wizard but we scrapped that.